airbyte-integrations/connectors/source-postgres/src/main/java/io/airbyte/integrations/source/postgres/ctid/CtidUtils.java

```diff
@@ -53,4 +60,19 @@ public record StreamsCategorised<T> (CtidStreams ctidStreams,
 
   }
 
+  /**
+   * Postgres servers version 14 and above are capable of running a tid scan.
+   * Used by ctid queries
+   * @param database database
+   * @return true for Tid scan capable server
+   */
+  public static boolean isTidScanCapableDBServer(final JdbcDatabase database) {
+    try {
+      return database.getMetaData().getDatabaseMajorVersion() >= POSTGRESQL_VERSION_TID_RANGE_SCAN_CAPABLE;
+    } catch (final SQLException e) {
+      LOGGER.warn("Failed to get db server version", e);
```

Comment:
Should we return `false` if an exception is thrown? Also, other connectors, such as MySQL, do not always throw `SQLException` - should we just catch a generic `Exception`?

Reply:
Since there were two and now 3 versions of postgres that _are_ capable to tid range scan, I opted for erring on the side of newer version. 
Do you think we should fall back and assume an older version?
I'll change to catch an `Exception`

Reply:
that makes sense returning true then.